ts/features/walletV3/payment/screens/WalletPaymentOutcomeScreen.tsx

```diff
@@ -1,28 +1,93 @@
-import { useNavigation } from "@react-navigation/native";
+import { Banner, VSpacer } from "@pagopa/io-app-design-system";
+import { openAuthenticationSession } from "@pagopa/io-react-native-login-utils";
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { RouteProp, useNavigation, useRoute } from "@react-navigation/native";
+import * as O from "fp-ts/lib/Option";
+import { pipe } from "fp-ts/lib/function";
 import React from "react";
+import { View } from "react-native";
 import { OperationResultScreenContent } from "../../../../components/screens/OperationResultScreenContent";
+import I18n from "../../../../i18n";
+import { mixpanelTrack } from "../../../../mixpanel";
 import {
   AppParamsList,
   IOStackNavigationProp
 } from "../../../../navigation/params/AppParamsList";
+import { useIOSelector } from "../../../../store/hooks";
+import { formatNumberCentsToAmount } from "../../../../utils/stringBuilder";
+import { WalletPaymentParamsList } from "../navigation/params";
+import { walletPaymentDetailsSelector } from "../store/selectors";
+import { WalletPaymentOutcome } from "../types/PaymentOutcomeEnum";
+
+type WalletPaymentOutcomeScreenNavigationParams = {
+  outcome: WalletPaymentOutcome;
+};
+
+type WalletPaymentOutcomeRouteProps = RouteProp<
+  WalletPaymentParamsList,
+  "WALLET_PAYMENT_OUTCOME"
+>;
 
 const WalletPaymentOutcomeScreen = () => {
+  const { params: _ } = useRoute<WalletPaymentOutcomeRouteProps>(); // TODO handle outcome
   const navigation = useNavigation<IOStackNavigationProp<AppParamsList>>();
+  const paymentDetailsPot = useIOSelector(walletPaymentDetailsSelector);
+
+  const paymentAmount = pipe(
+    paymentDetailsPot,
+    pot.toOption,
+    O.map(({ amount }) => formatNumberCentsToAmount(amount, true, "right")),
+    O.getOrElse(() => "--")
+  );
 
   const handleContinue = () => {
     navigation.popToTop();
     navigation.pop();
   };
+
+  const handleBannerPress = () => {
+    void mixpanelTrack("VOC_USER_EXIT", {
+      screen_name: "PAYMENT_OUTCOMECODE_MESSAGE"
+    });
+
+    return openAuthenticationSession(
+      "https://io.italia.it/diccilatua/ces-pagamento",
```

Comment:
Even if this line of code is not the subject of this PR, what do you think to extract this link as a constant into the constants file?

Reply:
You are right! I have moved the URL in a const inside ../utils. Let me know! https://github.com/pagopa/io-app/pull/5276/commits/f9f8339afba2dbd418943bcc60067e93bb7d80fb